sdk/storage/Azure.Storage.Common/swagger/Generator/package.json

```diff
@@ -20,7 +20,6 @@
     "@types/mocha": "5.2.5",
     "@types/node": "10.12.19",
     "@types/semver": "5.5.0",
-    "autorest": "^3.0.5231",
```

Comment:
Why are we removing this?

Reply:
Using autorest via npx didn't work so there is no need in local installation.